service/domain/graph/graph.go

```diff
@@ -28,31 +28,18 @@ func NewSocialGraph(local refs.Identity, hops Hops, storage Storage) (*SocialGra
 }
 
 func (g *SocialGraph) load(maxHops Hops, local refs.Identity, storage Storage) error {
-	localContact := feeds.MustNewContactFromHistory(local, true, false)
```

Comment:
This is all a small refactor to avoid creating a fake contact.

Reply:
Makes sense 👍